pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -153,34 +142,27 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
   const {
     mode,
     targets,
+    sources,
     customRulesFiles,
     withKnown,
     includedPackages,
     excludedPackages,
+    excludedRulesTags,
   } = getValues();
 
+  const methods = useForm();
+
   const steps = [
     {
       name: "Configure analysis",
       steps: [
         {
           name: "Analysis mode",
-          component: (
-            <SetMode
-              register={register}
-              mode={mode}
-              setMode={(val) => setValue("mode", val)}
-            />
-          ),
+          component: <SetMode mode={mode} />,
         },
         {
           name: "Set targets",
-          component: (
-            <SetTargets
-              targets={targets}
-              setTargets={(val) => setValue("targets", val)}
-            />
-          ),
+          component: <SetTargets targets={targets} />,
```

Comment:
Looking much better! Can we pull the actual form values from the context too to avoid this prop bubbling  overhead? Os there a way to type the form values pulled in from the context api?